include/coap2/coap_debug.h

```diff
@@ -36,8 +36,14 @@
  * Logging type.  One of LOG_* from @b syslog.
  */
 typedef short coap_log_t;
+/*
+   LOG_DEBUG+2 gives ciphers in GnuTLS
+   Use COAP_LOG_CIPHERS to output Cipher Info in OpenSSL etc.
+ */
+#define COAP_LOG_CIPHERS LOG_DEBUG+2
```

Comment:
Please add parentheses around `LOG_DEBUG+2`. Just to be safe...

Reply:
Updated and pushed